presets/4.3/tune/mouseFPV_AOS_35_tune_filters.txt

```diff
@@ -0,0 +1,168 @@
+#$ TITLE:  AOS 3.5 V1 Tune | mouseFPV
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
```

Comment:
make experimental plz, as all the fresh added presets getting status experimental